src/core/mesh/qgsmeshsimplificationsettings.h

```diff
@@ -0,0 +1,80 @@
+/***************************************************************************
+                         qgsmeshsimplificationsettings.h
+                         ---------------------
+    begin                : February 2020
+    copyright            : (C) 2020 by Vincent Cloarec
+    email                : vcloarec at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSMESHSIMPLIFICATONSETTINGS_H
+#define QGSMESHSIMPLIFICATONSETTINGS_H
+
+#include <QDomElement>
+
+#include "qgis_core.h"
+#include "qgis.h"
+
+/**
+ * \ingroup core
+ *
+ * Represents a overview renderer settings
+ *
+ * \note The API is considered EXPERIMENTAL and can be changed without a notice
```

Comment:
If this is the case, you could consider hiding it from Python for now?

Reply:
yes no need now in Python, so I add SIP_NO_FILE